### ISSUE TITLE
Cannot autocraft in AE2 crafting terminal

### ISSUE THREAD

**Issue by Batman1965** [Darkosto/SkyFactory4Issues#1033]
I cannot autocraft from any AE2 crafting terminal with either the Wireless Terminal or the Extra Cells 2 Wireless Universal Terminal in my hot bar. I can auto craft from both the Wireless Terminal and the Wireless Universal Terminal in this instance.

If I move the both the Wireless Terminal and the Wireless Universal Terminal from my hot bar to my inventory, or store them in my AE2 system, I am able to autocraft from an AE2 crafting terminal.

**Comment by Darkosto** [comment]
Just tested this: I was able to craft from the wireless terminal and the Crafting Terminal from AE2. This mechanic is working properly. I would suggest making sure you are not running out of power in you AE2 system, adding AE2 power cells can help alleviate issues.

**Comment by Batman1965** [comment]
I realize you’ve closed the issue but I’ve got plenty of power going to my AE system and I have 4 dense energy cells. 

I can autocraft from the wireless with zero issues. With the Crafting Terminal, I can auto craft as long as the wireless is not in my hot bar. With it in my hot bar, I can select the amount of items, but when I click next, it closes the dialog window rather than presenting me with the crafting plan window. This behaviour is consistent. I have tried it with both the default wireless terminal and the universal wireless terminal with the exact same results; with either wireless in my hot bar, I cannot get past the first auto crafting screen. I have tried reinstalling the mod pack; this does not fix the issue for me.
